### DISCUSSION TITLE
Posting a player.vimeo.com link doesn't create an embedded video player

### DISCUSSION THREAD

**Discussion by rv888** [mastodon/mastodon#25344] [Q&A]
I have a bunch of private videos that I can generate public urls through the Vimeo api and then use where ever I need. These links embed on Twitter, Facebook, Nostr, and other places just fine, but for some reason will not create the embed player on Mastodon. I am unable to use the standard vimeo.com/(video id) structure. When I post those, it does work but unfortunately I can't use those due to content licensing setup

Example link: http://player.vimeo.com/progressive_redirect/playback/223177818/rendition/360p/file.mp4?loc=external&oauth2_token_id=455795673&signature=9a88aac6ca2f5ada7ca7efc130d236e38680a87e0111708ac9a6aa11af8b4754

What am I missing?

Thanks!

**Comment by renchap** [comment]
This looks like a bug, I opened an issue: #25358
